CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - Default to OCI for image metadata and product image selection ([#561]).
+- Increase default memory reservation to `768Mi` to avoid OOMKilled ([#578]).
```

Comment:
One could argue it's breaking, as it enlarges the resource requirements of the cluster/namespace. No strong opinion though.
```suggestion
- Increase default memory reservation to `768Mi` to avoid `OOMKilled` ([#578]).
```